### PR TITLE
feat: handle orgs sublinks (i.e: work.ubq.fi/ubiquity)

### LINKED ISSUE #128: Add an organization filtering component
@0x4007 I know this was discussed previously and the "text field" can handle searching for a repo, but as an avid user of `work.ubq.fi` the lack of easy org filters actually bothers me a lot. I have implemented a pretty solution in around 4 hours, please give it a look.

[orgsSelector.webm](https://github.com/user-attachments/assets/57013b8f-174b-433d-b12e-34685b5ad37b)

If you like it lmk so I can put in more effort to polish and wrap it up. Open to feedbacks.

### PR DESCRIPTION
**Pull request by zugdev**
Resolves #128

Wait for #136 to be merged before merging this one since it uses PWA.

### CODE REVIEW

File: static/progressive-web-app.js [hunk 1]
```diff
@@ -47,6 +47,25 @@ self.addEventListener("activate", (event) => {
 self.addEventListener("fetch", (event) => {
   const url = new URL(event.request.url);
 
+  // Network-first strategy for index.html and dynamic routes
+  if (url.origin === location.origin && !urlsToCache.includes(url.pathname)) {
+    event.respondWith(
+      fetch(event.request)
+        .then((networkResponse) => {
```
Comment (0x4007) [inline static/progressive-web-app.js]: Can you change this to async await syntax

Comment (zugdev) [reply static/progressive-web-app.js]: will fix that once #136 goes through, since it changes PWA a lil bit

Comment (0x4007) [reply static/progressive-web-app.js]: Okay merged!

**Comment by zugdev** [comment]
The org filtering URL works even offline, I made it this way so eventually partners can have their own dev pool apps.

File: .gitignore [hunk 2]
```diff
@@ -8,6 +8,7 @@ node_modules
 .pnp.loader.mjs
 static/dist
 .env
+.wrangler/
```
Comment (0x4007) [inline .gitignore:11]: You sure about this?

File: static/progressive-web-app.js [hunk 3]
```diff
@@ -14,75 +14,87 @@ const urlsToCache = [
...
+        console.error("[Service Worker] Cache failed:", error);
+      }
+      self.skipWaiting(); // Activate the new worker immediately
+    })()
   );
-  self.skipWaiting(); // activate the new worker immediately
 });
 
 // Activate event (deletes old caches when updated)
 self.addEventListener("activate", (event) => {
```
Comment (0x4007) [inline static/progressive-web-app.js:31]: You can make this callback async instead I think it is cleaner.

**Review by 0x4007 — APPROVED** [review]

**Pull request merged**